From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tri Vo <trong@google.com>
Date: Mon, 20 May 2019 11:54:22 -0700
Subject: ANDROID: sync defconfigs with savedefconfig

Test: boot gki_defconfig and cuttlefish_defconfig on arm64 cuttlefish
Signed-off-by: Tri Vo <trong@google.com>
Change-Id: Ie0b518c0fd5100a39312069fe3b193684feb634d
---
 arch/arm64/configs/gki_defconfig | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 04f531ef2511..c9720858fd9a 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -35,13 +35,13 @@ CONFIG_PROFILING=y
 CONFIG_SCHED_MC=y
 CONFIG_SECCOMP=y
 CONFIG_PARAVIRT=y
+CONFIG_COMPAT=y
 CONFIG_ARMV8_DEPRECATED=y
 CONFIG_SWP_EMULATION=y
 CONFIG_CP15_BARRIER_EMULATION=y
 CONFIG_SETEND_EMULATION=y
 CONFIG_RANDOMIZE_BASE=y
 # CONFIG_EFI is not set
-CONFIG_COMPAT=y
 CONFIG_PM_WAKELOCKS=y
 CONFIG_PM_WAKELOCKS_LIMIT=0
 # CONFIG_PM_WAKELOCKS_GC is not set
@@ -81,7 +81,6 @@ CONFIG_IP_MULTIPLE_TABLES=y
 CONFIG_NET_IPGRE_DEMUX=y
 CONFIG_NET_IPVTI=y
 CONFIG_INET_ESP=y
-# CONFIG_INET_XFRM_MODE_BEET is not set
 CONFIG_INET_UDP_DIAG=y
 CONFIG_INET_DIAG_DESTROY=y
 CONFIG_IPV6_ROUTER_PREF=y
@@ -180,7 +179,6 @@ CONFIG_MAC80211=y
 CONFIG_RFKILL=y
 CONFIG_PCI=y
 CONFIG_PCI_HOST_GENERIC=y
-# CONFIG_UEVENT_HELPER is not set
 # CONFIG_ALLOW_DEV_COREDUMP is not set
 CONFIG_DEBUG_DEVRES=y
 CONFIG_ZRAM=y
@@ -282,7 +280,6 @@ CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_DRM=y
 # CONFIG_DRM_FBDEV_EMULATION is not set
 CONFIG_DRM_VIRTIO_GPU=y
-CONFIG_BACKLIGHT_LCD_SUPPORT=y
 # CONFIG_LCD_CLASS_DEVICE is not set
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
 CONFIG_SOUND=y
